train.py

```diff
@@ -768,6 +792,8 @@ def get_train_test_files(prefix, foldnums, allfolds, reduced, prefix2):
         i = 'all'
         files[i] = {}
         files[i]['train'] = files[i]['test'] = '%s.types' % prefix
+        if bool(percent_reduced):
+            files[i]['reduced_train'] = files[i]['reduced_test'] = '%s.types' % prefix
         if reduced:
```

Comment:
should be elif?

Reply:
Once this is called it has already passed the check that only one of bool(percent_reduced) and reduced is set to True, and I wanted to keep the structure of the code as separate if statements intact. The separate if statements and an elif would output the same result.